src/org/zaproxy/zap/extension/ascanrules/TestPersistentXSSAttack.java

```diff
@@ -126,16 +145,22 @@ public void init() {
 			return null;
 		}
 
-		HtmlContextAnalyser hca = new HtmlContextAnalyser(sinkMsg2);
+		HtmlContextAnalyser hca = new HtmlContextAnalyser(sourceMsg2);

```

Comment:
Was this change on purpose?

Reply:
Probably overzealous copy/pasting. Will review.